admin/views/hdf/confirm-and-submit.ejs

```diff
@@ -1,78 +1,77 @@
+<main class="govuk-main-wrapper" id="main-content" role="main">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <form method="post" action="/attendance/confirm-and-submit">
+                <input type="hidden" name="_csrf" value="<%= csrftoken %>" />
+                <% if (error.hasError()) { %>
+                <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="govuk-error-summary">
+                    <h2 class="govuk-error-summary__title" id="error-summary-title">
+                        You need to fix the errors on this page before continuing.
+                    </h2>
+                    <div class="govuk-error-summary__body">
+                        <p>See highlighted errors below.</p>
+                        <ul class="govuk-list govuk-error-summary__list">
+                        <% error.getFields().map(f => { %>
+                            <% if (typeof error.get(f) !== 'boolean') { %>
+                                <li><a href="#<%= f %>"><%= error.get(f) %></a></li>
+                            <% } %>
+                        <% }); %>
+                        </ul>
+                    </div>
                 </div>
-            </div>
-            <div class="form-group inset <%= error.isError('confirmBoxes') ? 'form-group-error' : '' %>" id="confirm">
-                <% if (error.isError('confirmBoxes')) { %>
-                    <span class="error-message"><%= error.get('confirmBoxes') %></span>
                 <% } %>
-                <div class="multiple-choice smaller">
-                    <input type="checkbox" id="pupilDetails" name="pupilDetails" value="checked" <% if (formData['pupilDetails']) {%> checked<% } %>>
-                    <label for="pupilDetails">pupil details are correct</label>
-                </div>
-                <div class="multiple-choice smaller">
-                    <input type="checkbox" id="uniquePins" name="uniquePins" value="checked" <% if (formData['uniquePins']) {%> checked<% } %>>
-                    <label for="uniquePins">each pupil who took the check used the unique PIN generated for them</label>
+                <h1 class="govuk-heading-xl">Confirm and submit</h1>
+                <%- partial('partials/_gds_session_expiry') %>
+                <div class="govuk-form-group" id="confirm">
+                    <div class="multiple-choice">
+                        <input class="govuk-radios__input" id="confirmYes" type="radio" role="radio" name="confirm" value="Y" checked="checked">
+                        <label for="confirmYes">
+                            <p>I <strong>confirm</strong> that my school administered the multiplication tables check according to the arrangements set out in the guidance.</p>
+                            In particular:
+                            <a id="confirmBoxes"></a>
```

Comment:
This looks like a good candidate for a button element

Reply:
@jon-shipley nope, this was added by Dan to navigate from the error box at the top to the part of the page where the error occurs (in our case the checkboxes right below)